dotnet/src/SemanticKernel.IntegrationTests/TemplateLanguage/tests.txt

```diff
@@ -58,8 +58,8 @@ ERROR
 {{ "{{" }} and {{ "}}" }} x
 {{ and }} x
 
- {{ " nothing special about these sequences: \ \0 \n \t \r \foo" }} 
-  nothing special about these sequences: \ \0 \n \t \r \foo 
+ {{ " nothing special about these sequences: \ \0 \n \t \r \foo" }}
+  nothing special about these sequences: \ \0 \n \t \r \foo
```

Comment:
do not edit this file pls, see instructions at the top

Reply:
can we change the script not to edit json and txt files?

Reply:
Good call. Trying out new tooling, but I'll disable the EOF newline rule.